.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.vscode/
```

Comment:
- [ ] Kindly add button interactions (hover, pressed, enabled) for all the buttons in the portfolio.
<img width="389" alt="Screenshot 2022-08-20 at 9 45 29 PM" src="https://user-images.githubusercontent.com/70269507/185761889-d92b737d-82fd-4e52-9d7f-4a0876ac721f.png">

- [ ] Kindle use a grid for the works section as per the requirements.

- [ ] Kindly for the technology categories ("Languages", "Frameworks" and "Skills") make sure they are in a ul, each of them in a li.

- [ ] Kindly for the technology in each category(e.g., "JavasScript", "HTML" and "CSS") make sure they are also a list ul and each of them a li.